specs/scenarios/trust-store-and-trust-policy.md

```diff
@@ -0,0 +1,212 @@
+- Wabbit Networks Kubernetes (WNK) is the team managing the *platform layer* (i.e. the Kubernetes cluster) within the Wabbit Networks company
+- Wabbit Networks Batch (WNB) team is the team managing the *service layer* (i.e. the Batch service) within the Wabit Networks company
+
+As usual ACME Rockets (ACME) is the customer that deploys the Batch applications and they own the *application layer*.
+
+Quite often, different teams in the enterprise have separate container registries for storing the images they need for their services. To deploy the Kubernetes cluster, the WNK team manages a set of images in the `k8s.wabbitnetworkscr.io` registry. To deploy the Batch service, the WNB team manages a set of images in the `batch.wabbitnetworkscr.io`. And, of course, the customer ACME manages their application images in the `mr.acmecr.io` registry.
+
+Furthermore, for security reasons, the teams within Wabbit Networks sign the images with their own keys devided from the Wabbit Networks root. ACME's team signs their images with a key that is derived from the ACME root key.
+
+## Registry to Signing Key Mapping
+Here is the mapping between registries and keys based on the above:
+
+| root | signing key | registry |
+| ----- | ----- | ----- |
+| Wabbit Networks | Wabbit Networks Kubernetes private key | `k8s.wabbitnetworkscr.io` |
+| Wabbit Networks | Wabbit Networks Batch private key | `batch.wabbitnetworkscr.io` |
+| ACME Rockets | ACME Rockets Map/Reduce private key | `mr.acmecr.io` |
+
+## Scenario: Trust for Multi-Tenant Services
+With the background from the above, this section will describe the trust scenario for multi-tenant services.
+
+### Delivering the Trust Store and Trust Policy for the Plaform Layer
+A Kubernetes node is created (either when a new cluster is created or an existing one is expanded) from an existing VM image. For Kubernetes management plane to be able to pull the platform containers, it needs to have the trust certificates already available on the node. Therefore, the VM image needs to deliver the trust certificates for the *platform layer* as part of the VM image itself. 
+
+In our case, the VM image should already have the *trust store* created on the VM file system and have the certificates for the *platform layer* available on the file system. Here is how the *trust store* should look like when the node is spun up.
+
+```
+$XDG_CONFIG_HOME/notation/trust-store
+    /x509
+        /ca
+            /wabbit-networks
+                wabbit-networks-kubernetes.crt
+        /tsa
+            /publicly-trusted-tsa
+                tsa-cert1.pem
+```
+However, just the *trust store* is not enough for Notation to validate the containers used for the *platform layer*. A *trust policy* needs to be established for the *platform layer* containers. Here is how the trust policy needs to look for a brand new node in a Kubernetes cluster based on the above example.
```

Comment:
Where is notation installed?

Reply:
Can you elaborate how the Notation installation location is relevant for the store structure?

Reply:
Yes, it is not relevant to the scenario. No need to discuss details. I am just curious about how notation is installed. Notation here could mean notation CLI or notation API.